pkg/noderesourcetopology/plugin.go

```diff
@@ -106,6 +108,16 @@ func (tm *TopologyMatch) Name() string {
 	return Name
 }
 
+var (
+	// once guards singletonNodeTopologyLister to be initialized only once.
+	once sync.Once
+	// singletonNodeTopologyLister is a shared NRT lister across different NRT plugin instances.
+	// It could happen while NRT plugin is registered in multiple profiles.
+	singletonNodeTopologyLister listerv1alpha2.NodeResourceTopologyLister
```

Comment:
@NoicFank would you on-hold this change for a while? as another PR is refactoring the code use controller-runtime client, and once it's merged, this lister is no longer needed.

Reply:
Okay! maybe I need to close this PR?

BTW, does using dynamic client may cause performance issues with scheduling, compared to using CRD informer?

Reply:
for lister, it shouldn't - it's also retrieving objs from in-memory cache.

Reply:
Ok, that's great.